htmx-pwa-frontend/index.ts

```diff
@@ -0,0 +1,28 @@
+import path from 'node:path';
+import { Entry } from './app/main';
+import { boilerplate } from './lib/';
+import assert from 'node:assert';
+
+export const build = async (): Promise<{
```

Comment:
this is a weird file.... imo is it htmx specific?

Reply:
Yeah it is a little weird. I wanted to make sure files were only omitted as a result of the root packagejson and not the inner front-end package